src/modules/search/components/ResultsFilter.jsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+import styled from 'styled-components';
+const scIcon = require('./images/soundcloud.svg');
+const ytIcon = require('./images/youtube.svg');
+
+export default function ResultsFilter(props) {
+	const choices = [
+		{name: 'YoutubeResult', icon: ytIcon},
+		{name: 'SoundCloudResult', icon: scIcon},
+		{name: 'both'}
+	];
+	return (
+		<Filter>
+			{choices.map(choice => {
+				return (
+					<Option>
```

Comment:
We're going to need to add a `key` prop to `Option`. While we're at it, we can just make this body an expression (kill the brackets and the `return`)